pynautobot/core/api.py

```diff
@@ -110,15 +110,13 @@ def __init__(
         self.users = App(self, "users")
         self.plugins = PluginsApp(self)
         self.graphql = GraphQLQuery(self)
-        self._validate_version()
```

Comment:
By removing this in the init, we no longer validate the version unless the end user explicitly calls the property themselves now (barring a single method that I could find). Can we make every call get the `.version` property so it will trigger the `_validate_version` method on the first one and just return the cached property on every other one?

Reply:
@brandomando on this one...